src/main/java/online/umbcraft/libraries/utils/MaterialUtil.java

```diff
@@ -0,0 +1,20 @@
+package online.umbcraft.libraries.utils;
+
+import online.umbcraft.libraries.GoldenDupes;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+
+public class MaterialUtil {
+
+    public static boolean isTotem(Material m) {
+        return m == ((GoldenDupes.getInstance().serverVersion() <= 12) ? Material.valueOf("TOTEM") : Material.TOTEM_OF_UNDYING);
+    }
+
+    public static boolean isAnvil(Material m) {
+        return m == ((GoldenDupes.getInstance().serverVersion() <= 12) ? Material.valueOf("ANVIL") : Material.DAMAGED_ANVIL);
```

Comment:
Is this enough to tell if the anvil is in its damaged state when on version<=1.12? To me it seems like this just checks if the block is any anvil, not specifically a damaged anvil. I believe you have to check the block data value as well to correctly support older versions.
(for example, in 1.8 jungle planks are `minecraft:planks:3`, similarly damaged anvils were `minecraft:anvil:2`)

Reply:
Yes, this only checks the Material is anvil, and need to add getData check to see whether a damaged_anvil. I didn't consider this before.